pages/_app.tsx

```diff
@@ -26,6 +26,18 @@ export default function App({ Component, pageProps }: AppProps) {
     router.push('/');
   };
 
+  const handleEditRecipe = (recipeToUpdate: Recipe) => {
+    setRecipes((recipes) => {
+      return recipes.map((recipe) => {
+        if (recipe.id === recipeToUpdate.id) {
+          return recipeToUpdate;
+        } else {
+          return recipe;
+        }
+      });
+    });
+  };
+
```

Comment:
I am not sure, but I think you could write it in an more compact way. Maybe I'm wrong. Good work though!

Reply:
Thanks, you're right 👍 
I now use a ternary to write this and use way less characters by using it 💪